zerver/lib/str_utils.py

```diff
@@ -8,7 +8,7 @@
 1.  text strings: These strings are used to represent all textual data,
     like people's names, stream names, content of messages, etc.
     These strings can contain non-ASCII characters, so its type should be
-    typing.Text (which is `str` in python 3 and `unicode` in python 2).
+    typing.str (which is `str` in python 3 and `unicode` in python 2).
```

Comment:
I would like to get your attention to this specific line @timabbott 
Better say this specific file. Although I think the changes in the file should make sense given we execute them in python 3 only yet I think there is modification needed to this comment. I am not sure how should we modify the comment since there is no python 2 now?

Reply:
I think we're going to end up removing that file soon, so it doesn't really matter.